file-formats/uipg/type/zif_aff_uipg_v1.intf.abap

```diff
@@ -0,0 +1,169 @@
+
+  CONSTANTS:
+    "! <p class="shorttext">Tile Display Format</p>
+    "! Tile display format
+    BEGIN OF co_tile_display_format,
+      "! <p class="shorttext">Standard Tile</p>
+      "! Standard tile
+      "! $enumValue 'TI'
+      standard  TYPE ty_tile_display_format VALUE 'TI',
+      "! <p class="shorttext">Wide Tile</p>
+      "! Wide tile
+      "! $enumValue 'TW'
+      wide      TYPE ty_tile_display_format VALUE 'TW',
+      "! <p class="shorttext">Flat Tile</p>
+      "! Flat tile
+      "! $enumValue 'FT'
+      flat      TYPE ty_tile_display_format VALUE 'FT',
+      "! <p class="shorttext">Flat Wide</p>
+      "! Flat wide
+      "! $enumValue 'FW'
+      flat_wide TYPE ty_tile_display_format VALUE 'FW',
+      "! <p class="shorttext">Link</p>
+      "! link
+      "! $enumValue 'LK'
+      link      TYPE ty_tile_display_format VALUE 'LK',
+    END OF co_tile_display_format.
+
+  TYPES:
+    "! <p class="shorttext">Page Section</p>
+    "! Page Section
+    BEGIN OF ty_tile_assignment,
+      "! <p class="shorttext">Tile Catalog Id</p>
+      "! Tile catalog id
+      tile_catalog_id     TYPE c LENGTH 35,
+      "! <p class="shorttext">Tile Catalog Type</p>
+      "! Tile catalog type
+      tile_catalog_type   TYPE ty_catalog_type,
+      "! <p class="shorttext">Tile Id</p>
+      "! Tile id
+      tile_id             TYPE c LENGTH 50,
+      "! <p class="shorttext">Tile Display Format</p>
+      "! tile display format
+      tile_display_format TYPE ty_tile_display_format,
+      "! <p class="shorttext">Target Mapping Catalog Id</p>
+      "! Target Mapping catalog id
+      tm_catalog_id       TYPE c LENGTH 35,
+      "! <p class="shorttext">Target Mapping Catalog Type</p>
+      "! Target Mapping catalog type
+      tm_catalog_type     TYPE ty_catalog_type,
+      "! <p class="shorttext">Target Mapping Id</p>
+      "! Target Mapping id
+      tm_id               TYPE c LENGTH 50,
+    END OF ty_tile_assignment.
+
+  "! <p class="shorttext">Item Assignment Type</p>
+  "! Item assignment type
+  "! $values {@link zif_aff_uipg_v1.data:co_item_type}
+  "! $default {@link zif_aff_uipg_v1.data:co_item_type.tile}
+  TYPES ty_item_type TYPE c LENGTH 2.
+
+  CONSTANTS:
+    "! <p class="shorttext">Item Assignment Type</p>
+    "! Item assignment type
+    BEGIN OF co_item_type,
+      "! <p class="shorttext">Tile</p>
+      "! Tile
+      "! $enumValue 'TI'
+      tile TYPE ty_item_type VALUE 'TI',
+    END OF co_item_type.
+
+  TYPES:
+    "! <p class="shorttext">Page Section Item Assignment</p>
+    "! Page Section Item Assignment
+    BEGIN OF ty_item,
+      "! <p class="shorttext">ID</p>
+      "! Id of the Assignment
+      id              TYPE c LENGTH 35,
+      "! <p class="shorttext">Type</p>
+      "! Type of the Assignment
+      type            TYPE ty_item_type,
+      "! <p class="shorttext">ID</p>
+      "! Id of the Page Section
+      tile_assignment TYPE ty_tile_assignment,
```

Comment:
The name of the component does not really match the title and the description. We highly recommend to have title and component name in sync